src/docs/12_glossary.adoc

```diff
@@ -23,9 +23,11 @@ Potentially more columns in case you need translations.
 
 ****
 
-[options="header"]
+[options="header", cols="0, 4"]
 |===
-| Term         | Definition
-| <Term-1>     | <definition-1>
-| <Term-2>     | <definition-2>
+| Term                      | Definition
+| [[pod_definition]]POD     | Personal Online Datastore
+| [[solid_definition]]SOLID | <definition-2>
```

Comment:
Can you help us defining these terms?

Reply:
Solid (Social Linked Data) is a web decentralization project led by Tim Berners-Lee, the inventor of the World Wide Web.